VmAgent.Core/Interfaces/ISessionHostRunner.cs

```diff
@@ -26,6 +26,5 @@ public interface ISessionHostRunner
 
         Task CollectLogs(string id, string logsFolder, ISessionHostManager sessionHostManager);
 
-        Task CreateStartGameExceptionLogs(string logFolder, string exceptionMessage);
```

Comment:
this is not used in DockerContainerEngine.cs so I removed this and updated the one in ProcessRunner.cs. Maybe this method initially had a purpose. Let me know if I missed something.

Reply:
It was done like that to avoid breaking something in the future: https://github.com/PlayFab/MpsAgent/pull/112/files#r910292900
But seeing how you redesigned the method, it should be fine